### DISCUSSION TITLE
Discord server?

### DISCUSSION THREAD

**Discussion by marcyvillegas** [excaliburjs/Excalibur#2520] [General]
It would be awesome if there is a discord server for this interesting library! We can make our own game jams etc.

**Comment by eonarheim** [comment]
Hi @marcyvillegas 

Thanks for the post! The Excalibur.js team does not have a chat community (Slack, Discord, IRC, etc.).

Running, managing, and moderating a service like that for Excalibur is currently beyond the scope of our available time and resources.
We encourage you to participate in the Discussions space here on GitHub.
